stable/locust/templates/master-deployment.yaml

```
@@ -141,24 +144,13 @@ spec:
           - containerPort: 5558
             name: loc-master-p2
             protocol: TCP
+        {{- if .Values.master.extraPorts -}}
+        {{ toYaml .Values.master.extraPorts | nindent 10 }}
+        {{- end }}
         readinessProbe:
-          initialDelaySeconds: 5
-          periodSeconds: 30
-          timeoutSeconds: 30
-          failureThreshold: 2
-{{- if .Values.loadtest.headless }}
-          tcpSocket:
-            port: 5557
-{{ else }}
-          httpGet:
-            path: /
-            port: 8089
-{{- if .Values.master.auth.enabled }}
-            httpHeaders:
-                - name: Authorization
-                  value: Basic {{ printf "%s:%s" .Values.master.auth.username .Values.master.auth.password | b64enc }}
-{{- end }}
-{{- end }}
+          {{- toYaml .Values.master.readinessProbe | nindent 12 }}
+        livenessProbe:
+          {{- toYaml .Values.master.livenessProbe | nindent 12 }}
```

For whatever reason, you gotta do this:

```suggestion
        {{- if .Values.master.livenessProbe }}
        livenessProbe:
          {{- toYaml .Values.master.livenessProbe | nindent 12 }}
        {{- end }}
```

---

Awesome. Great catch. Thank you!